lib/shared/layers/python-sdk/python/genai_core/models.py

```diff
@@ -55,8 +55,14 @@ def list_bedrock_models():
         if not bedrock:
             return None
 
-        response = bedrock.list_foundation_models()
-        bedrock_models = response.get("modelSummaries", [])
+        response = bedrock.list_foundation_models(
+            byInferenceType="ON_DEMAND", byOutputModality="TEXT"
```

Comment:
Can we leverage enums instead of hardcoded values?

https://github.com/aws-samples/aws-genai-llm-chatbot/blob/main/lib/shared/layers/python-sdk/python/genai_core/types.py

probably need 2 new enums for inference type and model life cycle status

Reply:
Would be nice if these enums were provided by boto3 :-)